src/main/cpp/RobotContainer.cpp

```diff
@@ -45,32 +46,49 @@ RobotContainer::RobotContainer() {
   // Set up default drive command
   // The left stick controls translation of the robot.
   // Turning is controlled by the X axis of the right stick.
+
   m_drive.SetDefaultCommand(frc2::RunCommand(
       [this] {
+        InputUtils::DeadzoneAxes axes = InputUtils::CalculateCircularDeadzone(
+            m_driverController.GetLeftX(), m_driverController.GetLeftY(),
+            OIConstants::kDriveDeadband);
         m_drive.Drive(
-            -units::meters_per_second_t{frc::ApplyDeadband(
-                m_driverController.GetLeftY(), OIConstants::kDriveDeadband)},
-            -units::meters_per_second_t{frc::ApplyDeadband(
-                m_driverController.GetLeftX(), OIConstants::kDriveDeadband)},
+            -units::meters_per_second_t{axes.y},
+            -units::meters_per_second_t{axes.x},
             -units::radians_per_second_t{frc::ApplyDeadband(
                 m_driverController.GetRightX(), OIConstants::kDriveDeadband)},
             true, false, kLoopTime);
       },
       {&m_drive}));
+#ifndef TEST_SWERVE_BOT
+  pathplanner::NamedCommands::registerCommand("LedFunni", m_leds.Intaking());
+  pathplanner::NamedCommands::registerCommand(
+      "Shoot Speaker",
+      ScoringCommands::Score([] { return ScoringDirection::Subwoofer; },
+                             &m_scoring, &m_intake));
+  pathplanner::NamedCommands::registerCommand(
+      "Intake", IntakingCommands::Intake(&m_intake));
+#endif
 
   // This won't work since we're getting the reference of an r-value which goes
   // out of scope at the end of the method
   m_chooser.SetDefaultOption("Leave Community", m_defaultAuto.get());
+  m_chooser.AddOption("3 in Amp", m_3inAmp.get());
+  m_chooser.AddOption("2 Note Auto", m_2noteAuto.get());
+  m_chooser.AddOption("4 Note Auto", m_4noteAuto.get());
+  m_chooser.AddOption("Kepler", m_kepler.get());
+  m_chooser.AddOption("Kepler2", m_kepler2.get());
   ShuffleboardLogger::getInstance().logVerbose("Auto Modes", &m_chooser);
-
-  // TODO: replace with a FUNNI animation
-  pathplanner::NamedCommands::registerCommand("LedFunni", m_leds.Intaking());
 }
 
 void RobotContainer::ConfigureButtonBindings() {
-  frc2::JoystickButton(&m_driverController,
-                       frc::XboxController::Button::kRightBumper)
-      .WhileTrue(new frc2::RunCommand([this] { m_drive.SetX(); }, {&m_drive}));
+  frc2::JoystickButton(&m_driverController, frc::XboxController::Button::kStart)
```

Comment:
2 things:
Why is this still the old way of binding instead of command-based?
Shouldn't this be OnTrue instead of WhileTrue?

Reply:
I believe that's because the binding came from the original MAXSwerve template